chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1521,19 +1505,19 @@ impl Actor for PeerManagerActor {
         }
 
         // Periodically push network information to client
-        self.push_network_info_trigger(ctx);
+        self.push_network_info_trigger(ctx, self.config.push_info_period);
 
         // Start peer monitoring.
```

Comment:
mention, that for this one, the waiting time differs (as AFAIK it uses some exponential backoff inside)

Reply:
@mm-near Can you take a look again. I updated description, and i added extra argument max_interval. This argument tells us maximum interval at which the trigger will run,